docs/node/run-your-node/keymanager-node/signing-key-manager-policy.md

```diff
@@ -0,0 +1,192 @@
+  --verbose
+```
+
+This should output something like the following:
+
+:::tip
+
+This is an example of an actual Oasis Testnet Key Manager policy.
+
+:::
+
+```bash
+{
+  "serial": 8,
+  "id": "4000000000000000000000000000000000000000000000004a1a53dff2ae482d",
+  "enclaves": {
+    "ZhD5ufyc/MReZD1qMSKNCRxnkNiZ3BtxqcYdx4+M0N9AJdq369ofvsxONjdgbgISFND0HG0EIv03iyqLiIGEWQ==": {
+      "may_query": {
+        "0000000000000000000000000000000000000000000000000000000000000000": [
+          "c0SidcKhBx3iuonmtXURnFB+qIVkg+nAiaAozAh16ltAJdq369ofvsxONjdgbgISFND0HG0EIv03iyqLiIGEWQ=="
+        ],
+        "000000000000000000000000000000000000000000000000a6d1e3ebf60dff6c": [
+          "LwbLEQ6dv+R5wv5q5CGRZWiEBWGxgCi/gpphcJFQ5zVAJdq369ofvsxONjdgbgISFND0HG0EIv03iyqLiIGEWQ=="
+        ]
+      },
+      "may_replicate": [
+        "jTX8etUcGSQBq3C4WbLlexga7dhQFnwzSJOEmRCPvfRAJdq369ofvsxONjdgbgISFND0HG0EIv03iyqLiIGEWQ=="
+      ]
+    },
+    "jTX8etUcGSQBq3C4WbLlexga7dhQFnwzSJOEmRCPvfRAJdq369ofvsxONjdgbgISFND0HG0EIv03iyqLiIGEWQ==": {
+      "may_query": {
+        "0000000000000000000000000000000000000000000000000000000000000000": [
+          "c0SidcKhBx3iuonmtXURnFB+qIVkg+nAiaAozAh16ltAJdq369ofvsxONjdgbgISFND0HG0EIv03iyqLiIGEWQ=="
+        ],
+        "000000000000000000000000000000000000000000000000a6d1e3ebf60dff6c": [
+          "LwbLEQ6dv+R5wv5q5CGRZWiEBWGxgCi/gpphcJFQ5zVAJdq369ofvsxONjdgbgISFND0HG0EIv03iyqLiIGEWQ=="
+        ]
+      },
+      "may_replicate": []
+    }
+  }
+}
+```
+
+The `"serial"` key, e.g. `8`, represents the key manager policy's serial number that must
+increase with every update of the key manager policy.
+
+The `"id"` key, e.g.
+`"4000000000000000000000000000000000000000000000004a1a53dff2ae482d"`, represents
+the key manager ParaTime's runtime id.
+
+The keys below `"enclaves"`, e.g.
+`"ZhD5ufyc/MReZD1qMSKNCRxnkNiZ3BtxqcYdx4+M0N9AJdq369ofvsxONjdgbgISFND0HG0EIv03iyqLiIGEWQ=="`
+and `"jTX8etUcGSQBq3C4WbLlexga7dhQFnwzSJOEmRCPvfRAJdq369ofvsxONjdgbgISFND0HG0EIv03iyqLiIGEWQ=="`,
+represent the identities of the key manager enclaves.
+Each key manager enclave id is comprised of two parts: its `MRENCLAVE` and its
+`MRSIGNER`.
+
+Each key manager enclave identity has two lists: `"may_query"` and
+`"may_replicate"`.
+
+Items in `"may_query"` list, e.g.
+`"0000000000000000000000000000000000000000000000000000000000000000"` and
+`"000000000000000000000000000000000000000000000000a6d1e3ebf60dff6c"`,
+represent the runtime ids of the ParaTimes that are allowed to query the key
+manager (in this example, the Cipher and the Sapphire ParaTimes running on the
+Testnet).
+
+The items under runtime ids of the ParaTimes, e.g.
+`"c0SidcKhBx3iuonmtXURnFB+qIVkg+nAiaAozAh16ltAJdq369ofvsxONjdgbgISFND0HG0EIv03iyqLiIGEWQ=="` and
+`"LwbLEQ6dv+R5wv5q5CGRZWiEBWGxgCi/gpphcJFQ5zVAJdq369ofvsxONjdgbgISFND0HG0EIv03iyqLiIGEWQ=="`,
+represent the identities of the runtime enclaves.
+Similarly to the key manager enclave id, each runtime enclave id is comprised of
```

Comment:
```suggestion
Similar to the key manager enclave ID each runtime enclave ID is comprised of
```

Reply:
I changed the id capitalization. For the other comment I think my previous version sounds a bit better.